src/stylesheets/screens/survey-list.scss

```diff
@@ -33,7 +32,13 @@
   }
 
   &__navigation {
+    position: relative;
     margin: rem(15px 0 0 auto);
+    bottom: rem(80px);
```

Comment:
<!--
  0 failure: 
  2 warning:  
	  Expected "bot..., Expected an empty...
  
  
  DangerID: danger-id-Danger;
  File: src/stylesheets/screens/survey-list.scss;
  Line: 37;
-->

- :warning: 
	  Expected "bottom" to come before "margin" in group "Positioning" (order/properties-order)
- :warning: Expected an empty line before property "bottom" (order/properties-order)